ext/openssl/ossl_x509ext.c

```diff
@@ -318,12 +320,17 @@ ossl_x509ext_initialize_copy(VALUE self, VALUE other)
     return self;
 }
 
+/*
+ * call-seq:
+ *    extension.oid = string => string
```

Comment:
I think we generally don't show the return value (` => string`) for a setter method.